controllers/snapshot/snapshot_adapter.go

```diff
@@ -42,6 +43,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	appStudioGroupName  = "appstudio.redhat.com"
+	environmentKindName = "environment"
+)
+
```

Comment:
Wouldn't be better to create our custom error? We don't need this data for, and I doesn't like hardocoding those values.

Also `NotFound` is related to go client and we are not even using it, would be better to have error with better semantics `EnvironmentNotInNamespaceError`

Reply:
Updated. Thanks.